### PR TITLE
[BG-7182] Generate hardened child keys as master keys

### PR DESCRIPTION
**Pull request by dacruz21**
DAI requested that we automate the generation of master keys as hardened child keys of a single xprv. Added an admin tool command ``admin.js generate`` which generates a large number of private keys and stores them to a file. These private keys are imported with the import tool as before.

### CODE REVIEW

File: app/krs.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ exports.provisionKey = co(function *(req) {
   }
 
   key.masterKey = masterKey.xpub;
-  key.path = `m/${masterKey.keyCount}`;
+  key.path = masterKey.keyCount;
```
Comment (marktoda) [inline app/krs.js]: Var names are mismatched now, either this should be called key.keyCount or should keep the m/xxx with the new framework.

Comment (dacruz21) [reply app/krs.js]: Here's the issue: The key from which we derive wallet keys is no longer an `m` key, `m` is now a single master key from which multiple `m/k'` are derived. So the complete derivation path of a wallet key is now `m/k'/i` with k being the index of the hardened subkey, and i being the index of the wallet. We only need to store the value of `i` as the prime prevents us from deriving the entire xpub from `m`. Open to any suggestions on making this more clear.

Comment (marktoda) [reply app/krs.js]: So why not change the name to key.walletIndex or something, since it's not necessarily the path of the key anymore?

Comment (dacruz21) [reply app/krs.js]: Changed to index.

**Pull request merged**